packages/phoenix-evals/src/phoenix/evals/models/litellm.py

```diff
@@ -65,7 +65,7 @@ def _init_environment(self) -> None:
             from litellm import validate_environment
 
             self._litellm = litellm
-            env_info = validate_environment(self._litellm.utils.get_llm_provider(self.model))
```

Comment:
is this no longer needed?

Reply:
it looks like 
https://github.com/BerriAI/litellm/blob/6c79bea17791bf5781f6d656f641eadcc41dcf39/litellm/utils.py#L5588
however, it means, we need to bump `litellm` up to [v1.28.9](https://github.com/BerriAI/litellm/releases/tag/v1.28.9) in pyproject.toml.

Reply:
Let's revert this change if it's not strictly necessary? I don't think we need to bump the version for this

Reply:
Actually I think it's probably a good idea to bump the version, I can do that on my end, thanks so much for the PR @mkhludnev!